app/summary/summary.html

```diff
@@ -1,66 +1,55 @@
-
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1" />
-	<!--<meta name="description" content="Neon Admin Panel" />
-	<meta name="author" content="" />-->
-
-	<link rel="icon" href="/images/favicon.ico">
-
-	<title>IPA Summary</title>
-
-	<!--<link rel="stylesheet" href="assets/js/jquery-ui/css/no-theme/jquery-ui-1.10.3.custom.min.css">-->
-	<link rel="stylesheet" href="/css/entypo.css">
-	<link rel="stylesheet" href="/css/lib.css">
-	<link rel="stylesheet" href="/css/neon-core.css">
-	<link rel="stylesheet" href="/css/neon-theme.css">
-	<link rel="stylesheet" href="/css/neon-forms.css">
-	<link rel="stylesheet" href="/css/timeline.css">
-	<link rel="stylesheet" href="/css/shared.css">
-	<link rel="stylesheet" href="/css/summary.css">
-	<link rel="stylesheet" href="/css/instructionalSupportSummary.css">
-
-	<!--<script src="assets/js/jquery-1.11.3.min.js"></script>-->
-
-	<!--[if lt IE 9]><script src="assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
-
-	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-	<!--[if lt IE 9]>
-		<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-		<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-	<![endif]-->
-
-
-</head>
-
-<body class="page-body ng-cloak" data-url="http://neon.dev" ng-app="summaryApp" ng-cloak ng-controller="SharedCtrl">
-	<base href="/summary/">
-
-	<div ng-if="loadingView" class="overlay">
-		<spinner class="vertical-center"></spinner>
-	</div>
-
-	<div class="page-container" collapsable-sidebar-container ng-view>
-		<!-- add class "sidebar-collapsed" to close sidebar by default, "chat-visible" to make chat appear always -->
-	</div>
-	<no-mobile-support></no-mobile-support>
-
-
-
-
-
-	<!-- Bottom scripts (common) -->
-	<script src="/js/lib.js"></script>
-	<script src="/js/sharedApp.js"></script>
-	<script src="/js/clientConfig.js"></script>
-	<script src="/js/sharedConfig.js"></script>
-	<script src="/js/snippets.js"></script>
-	<script src="/js/summaryApp.js"></script>
-
-</body>
-
-</html>
\ No newline at end of file
+	<head>
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+
+		<link rel="icon" href="/images/favicon.ico">
+
+		<title>IPA Summary</title>
+
+		<!--<link rel="stylesheet" href="assets/js/jquery-ui/css/no-theme/jquery-ui-1.10.3.custom.min.css">-->
+		<link rel="stylesheet" href="/css/entypo.css">
+		<link rel="stylesheet" href="/css/lib.css">
+		<link rel="stylesheet" href="/css/neon-core.css">
+		<link rel="stylesheet" href="/css/neon-theme.css">
+		<link rel="stylesheet" href="/css/neon-forms.css">
+		<link rel="stylesheet" href="/css/timeline.css">
+		<link rel="stylesheet" href="/css/shared.css">
+		<link rel="stylesheet" href="/css/summary.css">
+		<link rel="stylesheet" href="/css/instructionalSupportSummary.css">
```

Comment:
I suspect css/instructionalSupportSummary.css was added by accident in the past.